test/integration/imagestream_test.go

```diff
@@ -245,32 +245,3 @@ func TestImageStreamMappingCreate(t *testing.T) {
 	}
 
 }
-
-func TestImageStreamDelete(t *testing.T) {
```

Comment:
`ImageStreams` don't have special REST handling like most image APIs.  They're just straight generic etcd storage, so this test is boring.